NerdyNomads/Text-Savvy
# Ev/feat/copy link to clipboard

## What does this PR do?

- Add the ability to copy the link to the source of a text.
- Add the ability to add a link when manually adding text to the clipboard

### Screenshots (optional)

![image](https://user-images.githubusercontent.com/66437477/156876897-d40e4a9e-4d2b-4fdc-87a4-09997489a059.png)

### Steps to test/review (optional)
**Test 1:**
- Login
- Click the copy button on one of the cards:
![image](https://user-images.githubusercontent.com/66437477/156876679-8cc742c1-6fa4-476d-91cb-b2885491dd5b.png)
- A notification should pop up notifying you that the link has been copied to your clipboard.
- Paste into a new tab to see if the copy was successful.

**Test 2:**
- Click the "Add Text" card
- Click the chain icon in the new card that shows up
- A text box should show up allowing you to type in/paste a link.
- This is not yet connected to the database yet as I still have to work out how IDs are going to be assigned to new texts

Note that the link/source to already existing texts are non-editable, I don't know if this is the correct behavior for what we want but I just left it as non-editable for now.

## Checklist before merging
      - Reviewers
      - Assignee
      - Labels
      - Project
      - Milestone
      - Linked Issues
